### ISSUE TITLE
`<Model>CreateManyInput` can be empty if the table contains only relations and an ID field, creating GraphQL syntax errors

### ISSUE THREAD

**Issue by Tenrys** [maoosi/prisma-appsync#70]
Prisma Schema:

```prisma
model PostAuthor {
  uuid String @id @default(uuid())

  userUuid String?
  user     User?   @relation(fields: [userUuid], references: [uuid])

  agentUuid String?
  agent     Agent?  @relation(fields: [agentUuid], references: [uuid])
  Post      Post[]
}
```

Generated GraphQL:

```
input PostAuthorCreateManyInput {
            
}
```

Workaround is to make a nullable field, I guess? I'm doing this so I can have some sort of way to link different types of models on a same relationship, since Prisma does not currently support that feature. I imagine GraphQL unions get no use because of that, right?

**Comment by maoosi** [comment]
Hey @Tenrys, thanks for reporting the issue. Nothing wrong with your model declaration and the use case for it.

While waiting for a fix, one quick workaround would be to add a temporary field (that isn't of type `@id` and isn't a relation field). Not super clean, but it should do the trick and allow you to run `npx prisma generate` without issue. Then you can remove this temporary field after a fix is released.

```prisma
model PostAuthor {
    // ...
    tmp  String?
}
```

**Comment by Tenrys** [comment]
It's what I did, yeah! Thanks for the quick response, this project looks solid :) Just wish that Prisma itself had a little more support for some more advanced GraphQL / PostgreSQL features, but I'll try to find some workarounds and maybe ask for help if I really get stuck, if that's okay with you.

**Comment by maoosi** [comment]
Fix released in [1.0.0-rc.4](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-rc.4).
